source/index-management.rst

```diff
@@ -662,6 +662,12 @@ Common API Components
      * in a geo haystack index.
      */
     bucketSize: Int32;
+
+    /**
+     * Specifies the index to be a partial index and to only index the documents matching the
+     * provided filter.
+     */
+    partialFilterExpression: Document;
```

Comment:
The "index to be a partial index" phrase is a bit awkward.  And it should mention that it's optional, like the other fields do.  How about

"Optionally specifies a filter, making this a partial index.  Only documents that match the filter expression are included in the index."

That's not great either, though.  Any other ideas?